api/src/db/model/quota.ts

```diff
@@ -130,16 +131,16 @@ export default class QuotaModel extends Model {
         }
     }
 
-    async findForQuotaSize(quotaSize: QuotaSize): Promise<any> {
+    async findForQuotaSize(quotaSize: ProjectQuotaSize): Promise<any> {
         const query = {
             text: `
                 SELECT json_build_object(
                     'cpu', (SELECT row_to_json(d) FROM (SELECT cpu_requests AS "requests", cpu_limits AS "limits"
-                        FROM ref_quota WHERE id = '${quotaSize}') d),
+                        FROM ref_quota WHERE id = '${quotaSize.quotaCpuSize}') d),
```

Comment:
although not an actual sql injection issue, use the text, values pattern. Do not directly place values! This is a bad habit that will lead to SQL injection